### PR TITLE
Move Creatures/Characters to TokenTemplates as STI

### PR DESCRIPTION
**Pull request by jdbann**

### CODE REVIEW

File: db/migrate/20200525103646_move_characters_to_token_templates.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        id,
+        type,
+        campaign_id,
+        name,
+        size,
+        created_at,
+        updated_at
+      )
+      SELECT
+        id,
```
Comment (cpytel) [inline db/migrate/20200525103646_move_characters_to_token_templates.rb:14]: It's a good thing we used uuids!

File: app/models/token.rb [hunk 2]
```diff
@@ -22,20 +22,20 @@ def self.size_names
   end
 
   def name
-    read_attribute(:name).presence || tokenable.try(:name)
+    read_attribute(:name).presence || token_template.try(:name)
```
Comment (cpytel) [inline app/models/token.rb:25]: Maybe not in this PR, but to confirm, is the plan to just always pull the name from `TokenTemplate` and drop `Token#name`?

File: db/migrate/20200525103646_move_characters_to_token_templates.rb (hunk 1 above)
Comment (jdbann) [reply db/migrate/20200525103646_move_characters_to_token_templates.rb:14]: I'm wondering if there's ever an argument for anything else!

File: app/models/token.rb (hunk 2 above)
Comment (jdbann) [reply app/models/token.rb:25]: Yes, I think so

**Pull request merged**